lego/apps/users/models.py

```diff
@@ -299,8 +300,17 @@ def email_address(self):
     def profile_picture(self, value):
         self.picture = value
 
+    @PermissionsMixin.is_abakus_member.setter
+    def is_abakus_member(self, add_to_group):
+        if self.is_verified_student():
```

Comment:
what happens if the user isn't a verified student? it just ignores it? seems like this should either be validated somewhere else, or fail loudly here

Reply:
It should fail hard 😊. The main point of checking if the user is verified is to only allow active students to join Abakus, not regular Users.

We could do this validation in the serializer, like we do with usernames, and raise eventual exception there instead if that is preferred.